ExtremeValueProject/AffineTransformation.lean

```diff
@@ -126,7 +128,16 @@ lemma affine_continuousAt_of_continuousAt
     {F : CumulativeDistributionFunction} {x : ℝ} (F_cont : ContinuousAt F x)
     (A : orientationPreservingAffineEquiv) :
     ContinuousAt (A • F) ((A : ℝ ≃ᵃ[ℝ] ℝ) x) := by
-  sorry -- **Issue #6**
+  have ha:=(A : ℝ ≃ᵃ[ℝ] ℝ)⁻¹.continuous_of_finiteDimensional
+  let f:= fun x => (A : ℝ ≃ᵃ[ℝ] ℝ)⁻¹ x
+  have hf1: ContinuousAt f ((A : ℝ ≃ᵃ[ℝ] ℝ) x) := by exact Continuous.continuousAt ha
+  have h_eq: (A • F).toStieltjesFunction = F ∘ f:= by rfl
+  rw[h_eq]
+  have h_simp: f ((A : ℝ ≃ᵃ[ℝ] ℝ) x) = x := by
+    unfold f
+    exact (AffineEquiv.apply_eq_iff_eq_symm_apply (A : ℝ ≃ᵃ[ℝ] ℝ)⁻¹).mpr rfl
+  rw[← h_simp] at F_cont
+  exact ContinuousAt.comp F_cont hf1
```

Comment:
It is possible to remove a few parts that are not necessary in the final version. (I know those are helpful when finding the proof, but the final version can be made shorter without losing readability.) I also added some spaces around colons etc. to follow standard code style.
```suggestion
  have ha := (A : ℝ ≃ᵃ[ℝ] ℝ)⁻¹.continuous_of_finiteDimensional
  let f := fun x ↦ (A : ℝ ≃ᵃ[ℝ] ℝ)⁻¹ x
  rw [show (A • F).toStieltjesFunction = F ∘ f from rfl]
  have h_simp : f ((A : ℝ ≃ᵃ[ℝ] ℝ) x) = x := (AffineEquiv.apply_eq_iff_eq_symm_apply _).mpr rfl
  rw[← h_simp] at F_cont
  exact ContinuousAt.comp F_cont ha.continuousAt
```

Reply:
Thank you!

I added a few suggestions of minor edits. You should be able to accept them by just clicking.

Remember to also write "close #6" as in step 4 of [the contribution instructions](https://github.com/kkytola/ExtremeValueProject/blob/main/README.md). Then the issue gets closed once the pull-request gets merged.